discovery-provider/src/queries/get_route_metrics.py

```diff
@@ -26,7 +26,42 @@ def get_route_metrics(args):
         return _get_route_metrics(session, args)
 
 
+def _make_metrics_tuple(metric):
+    return {
+        'timestamp': int(time.mktime(metric.time.timetuple())),
```

Comment:
just to double check let's make sure this is identical to what we currently return (timezone etc.)

Reply:
👍